src/plugins/saved_objects_management/public/plugin.ts

```diff
@@ -31,7 +31,7 @@
 import { i18n } from '@osd/i18n';
 import { CoreSetup, CoreStart, Plugin } from 'src/core/public';
 
-import { WizardStart } from '../../wizard/public';
+import { WizardStart } from '../../vis_builder/public';
```

Comment:
Is there a plan to rename SVG's, interfaces and class names as well in a different task or will it be kept as it is?

Reply:
Yes, those will be changed in separate PRs.